### DISCUSSION TITLE
Allow users to place files of any size on the wiki.

### DISCUSSION THREAD

**Discussion by daeqs-one** [requarks/wiki#7323] [Error / Bug Report] [answered]
I understand that the task of a wiki is primarily to place and structure information. However, there are situations when it is necessary to attach attachments to this information, including large ones. Your wiki is an excellent solution, among other things, because it allows you to centrally store all information exclusively in a database. This is one of the few solutions of this kind. Also, understanding the possible consequences in terms of storing and archiving data in this form, we would still like to limit ourselves to a single solution, without deploying additional infrastructure in the form of a file storage.

**Comment by NGPixel** [comment]
You can already do so...

**Reply by daeqs-one** [reply]
I'm sorry to bring up this topic again, but experimentally we managed to find out the maximum size of uploaded files, it does not exceed 1 (one) Gigabytes. When I talked about large files, I meant really large files (many Gigabytes), and alas, without third-party solutions that weigh down the infrastructure and reduce data connectivity, it has not yet been possible to find a workaround. Perhaps in the future you will have the time and opportunity to meet such a strange wish. Anyway, thanks for what you're doing. It's a really great product.

**Reply by NGPixel** [reply, marked as answer]
This is getting into DB limitations and really out of scope of what the wiki is able to handle. 1GB is already a stretch so multi-GB is not something that will be supported in the future unfortunately.
